Cannot build the ndk immediately after cloning the repo

### Expected behavior
When I clone the application, I can run `make build` and build all the `.so` files

### Observed behavior
I get the following stack trace
```
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/maltz/Documents/workspace/bugsnag-android-ndk/build.gradle' line: 41

* What went wrong:
A problem occurred evaluating root project 'bugsnag-android-ndk'.
> Unable to generate signature for '/Users/maltz/Documents/workspace/bugsnag-android-ndk/ndk/build/outputs/aar/ndk-release.aar' as no signatory is available to sign

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
make: *** [build] Error 1
```

### Steps to reproduce
Clone repository, run `make build`

### Version
1.1.2

### Additional information
Nope!

#### Can't comment on Issues?
Some users have been unable to comment on Github issues when an [adblocker extension is enabled](https://docs.bugsnag.com/platforms/browsers/faq/#is-bugsnag-blocked-by-ad-blockers).
We recommend temporarily disabling the extension, or if that fails, contacting support@bugsnag.com.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -95,7 +95,7 @@ This process is a little ridiculous...
 -   Build and upload the new version
 
     ```shell
-    ./gradlew clean assemble && ./gradlew publish
+    ./gradlew clean assemble -Preleasing=true && ./gradlew publish -Preleasing=true
     ```
 
 -   "Promote" the release build on Maven Central

--- a/Makefile
+++ b/Makefile
@@ -34,4 +34,4 @@ endif
 	@git commit -am "Release v$(VERSION)"
 	@git tag v$(VERSION)
 	@git push origin master v$(VERSION)
-	@./gradlew clean assemble uploadArchives bintrayUpload
+	@./gradlew clean assemble uploadArchives bintrayUpload -Preleasing=true

--- a/build.gradle
+++ b/build.gradle
@@ -19,7 +19,6 @@ ext {
 
 apply plugin: 'maven-publish'
 apply plugin: 'com.jfrog.bintray'
-apply plugin: 'signing'
 
 
 def target = 'ndk/build/outputs/aar/ndk-release.aar'
@@ -36,11 +35,15 @@ model {
     }
 }
 
-signing {
-    if (file(target).exists())
-        sign file(target)
-    if (file(pomPath).exists())
-        sign file(pomPath)
+if (project.hasProperty("releasing")) {
+    apply plugin: 'signing'
+
+    signing {
+        if (file(target).exists())
+            sign file(target)
+        if (file(pomPath).exists())
+            sign file(pomPath)
+    }
 }
 
 publishing {

